src/NServiceBus.RavenDB/Internal/DocumentStoreManager.cs

```diff
@@ -0,0 +1,184 @@
+﻿namespace NServiceBus.RavenDB.Internal
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Configuration;
+    using System.Linq;
+    using NServiceBus.Logging;
+    using NServiceBus.Persistence;
+    using NServiceBus.Settings;
+    using Raven.Client;
+    using Raven.Client.Document;
+
+    static class DocumentStoreManager
```

Comment:
I agree with @davidboike. This needs to be fixed

On Thursday, February 4, 2016, David Boike <notifications@github.com> wrote:

> In src/NServiceBus.RavenDB/Internal/DocumentStoreManager.cs
> <https://github.com/Particular/NServiceBus.RavenDB/pull/143#discussion_r51830502>
> :
>
> > @@ -0,0 +1,184 @@
> > +﻿namespace NServiceBus.RavenDB.Internal
> > +{
> > +    using System;
> > +    using System.Collections.Generic;
> > +    using System.Configuration;
> > +    using System.Linq;
> > +    using NServiceBus.Logging;
> > +    using NServiceBus.Persistence;
> > +    using NServiceBus.Settings;
> > +    using Raven.Client;
> > +    using Raven.Client.Document;
> > +
> > +    static class DocumentStoreManager
>
> Added some tests for that. I don't know about a few dozen though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus.RavenDB/pull/143/files#r51830502>
> .
>